File: appmonet-sdk/appmonet-sdk-core/src/main/java/com/monet/bidder/AdType.java
package com.monet.bidder;

import androidx.annotation.NonNull;

/**
 * Created by jose on 8/29/17.
 */

public enum AdType {
  BANNER("banner"),
  INTERSTITIAL("interstitial"),
  NATIVE("native");
  private String type;

  AdType(String type) {
    this.type = type;
  }

  @NonNull @Override
  public String toString() {
    return type;
  }
}

File: appmonet-sdk-core-kmm/src/commonMain/kotlin/com/monet/AdType.kt
package com.monet

/**
 * Created by jose on 8/29/17.
 */
enum class AdType(private val type: String) {
  BANNER("banner"),
  INTERSTITIAL("interstitial"),
  NATIVE("native");

  override fun toString(): String {
    return type
  }
}